src/libraries/Common/src/System/Runtime/InteropServices/Variant.cs

```diff
@@ -170,9 +174,11 @@ public unsafe void CopyFromIndirect(object value)
                     *(uint*)this._typeUnion._unionTypes._byref = (uint)value;
                     break;
 
+#pragma warning disable 618
```

Comment:
What required this be added?  IIRC, 0618 is for using an obsoleted member.

Reply:
`ErrorWrapper` is obsolete. I'll add a comment. (Same with the other `*Wrapper` classes for COM).

Reply:
> ErrorWrapper is obsolete

Thanks.  My question was really, why wasn't this suppression necessary before?

Reply:
Ahh, it looks like this is because those wrapper classes are marked obsolete in the ref assembly for System.Runtime.InteropServices, but not the implementation. So when this file was only used by CoreLib and not going through the ref assembly, it was not considered obsolete.

Reply:
I see. Ok, thanks.